### PR TITLE
Kupreichik A hw8 all tasks

### PR DESCRIPTION
**Pull request by akupreychik**

### CODE REVIEW

File: homework/Kupreichik Anton/HomeWork_4/ex1_Animals/com.company/service/Bird.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package com.company.service;
```
Comment (javaGuruBY) [inline homework/Kupreichik Anton/HomeWork_4/ex1_Animals/com.company/service/Bird.java:1]: move to beans

File: homework/Kupreichik Anton/HomeWork_4/ex1_Animals/com.company/service/Cat.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    public String getNameOfOwner() {
+        return nameOfOwner;
+    }
+
+    @Override
+    public String getColor() {
+        return color;
+    }
+
+    @Override
```
Comment (javaGuruBY) [inline homework/Kupreichik Anton/HomeWork_4/ex1_Animals/com.company/service/Cat.java:22]: if behavior not changed, no need to be Overriden

**Review by javaGuruBY — APPROVED** [review]
minor comments, otherwise  lgfm.

**Review by TLittleProgrammer — APPROVED** [review]
lgfm

**Review by vladis1350 — APPROVED** [review]
lgfm

**Pull request merged**